axis.py

```diff
@@ -237,7 +237,7 @@ def __init__(self, _db, input_identifier_types=None,
         axis_360_api = axis_360_api or Axis360API(_db)
         super(Axis360BibliographicCoverageProvider, self).__init__(
             _db, axis_360_api, DataSource.AXIS_360,
-            workset_size=25, 
+            batch_size=25, 
```

Comment:
This requires a bunch of changes but I think it has to happen. Every other piece of code refers to this as a 'batch', not a 'workset'.